addon/components/paper/date-picker.js

```diff
@@ -0,0 +1,59 @@
+import moment         from 'moment';
+import { get }        from '@ember/object';
+import PaperTextField from
+  'ember-railio-form-components/components/paper/text-field';
+
+export default PaperTextField.extend({
+  layoutName: 'components/paper/input-field',
+  inputType:  'text',
+  lazy:       true,
+  dateFormat: 'DD-MM-YYYY',
+
+  keyDown(e) {
+    let addValue;
+    let value = moment(get(this, 'value'), get(this, 'dateFormat'));
+    let scale = e.shiftKey ? 'month' : 'day';
+
+    if (e.key === 'ArrowUp')   { addValue = 1; }
+    if (e.key === 'ArrowDown') { addValue = -1; }
+
+    if (addValue) {
+      e.preventDefault();
+      this.send('changed', value.add(addValue, scale).toDate(), e, true);
+    }
+
+    this._super(...arguments);
+  },
+
+  format(value) {
+    return value ? moment(value).format('DD-MM-YY') : '';
+  },
+
+  formatBeforeUpdate(value) {
```

Comment:
This function is basically doing serialisation right? I would rename it to reflect that, because `format` might be a confusing name for what it is actually doing.